boards/digilent/arty_a7/Kconfig.arty_a7

```diff
@@ -0,0 +1,12 @@
+# Digilent Arty board configuration
+
+# Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ARTY_A7_DESIGNSTART_FPGA_CORTEX_M1
```

Comment:
Nothing wrong on this line, but the change of board name requires an update here:
https://github.com/zephyrproject-rtos/zephyr/blob/8b2234de9dfaee21f8fe1b6dcd3d95ee2235e091/samples/drivers/spi_flash/src/main.c#L16-L17

Reply:
Fixed